rules/windows/builtin/security/win_security_tap_driver_installation.yml

```diff
@@ -4,7 +4,10 @@ related:
     - id: 8e4cf0e5-aa5d-4dc3-beff-dc26917744a9
       type: derived
 status: test
-description: Well-known TAP software installation. Possible preparation for data exfiltration using tunnelling techniques
+description: |
+    Detects well-known TAP driver service installation. This could be a sign of potential preparation for data exfiltration using tunnelling techniques.
```

Comment:
```suggestion
    Detects the installation of a well-known TAP driver service. This could be a sign of potential preparation for data exfiltration using tunnelling techniques.
```